webviews/components/header.tsx

```diff
@@ -66,50 +67,66 @@ function Title({ title, number, url, canEdit, isCurrentlyCheckedOut, isIssue, re
 				}
 			}}
 		>
-			<textarea name="text" style={{ width: '100%' }} defaultValue={currentTitle}></textarea>
+			<input type="text" style={{ width: '100%' }} defaultValue={title}></input>
 			<div className="form-actions">
 				<button className="secondary" onClick={() => setEditMode(false)}>
 					Cancel
 				</button>
 				<input type="submit" value="Update" />
 			</div>
 		</form>
-	) : (
-		<h2>
-			{currentTitle} <a href={url} title={url}>#{number}</a>
-		</h2>
 	);
 
-	return (
+	const displayTitle = (
 		<div className="overview-title">
-			<div className="title-and-edit">
-				{editableTitle}
-				<div className="block-select">
-					{/*
-				For whatever reason, triple click on a block element in MacOS will select everything in that element, *and* every `user-select: false` block adjacent to that element.
-				Add an empty selectable div here to block triple click on title from selecting the following buttons. Issue #628.
-				*/}
-				</div>
-				{canEdit && !inEditMode ? (
-					<div className="flex-action-bar comment-actions">
-						{
-							<button title="Edit" onClick={() => setEditMode(true)}>
-								{editIcon}
-							</button>
-						}
-						{
-							<button title="Copy Link" onClick={copyPrLink}>
-								{copyIcon}
-							</button>
-						}
+			<h2>
+				{title}{' '}
+				<a href={url} title={url}>
+					#{number}
+				</a>
+			</h2>
+		</div>
+	);
+
+	const editableTitle = inEditMode ? titleForm : displayTitle;
+	return editableTitle;
+}
+
+function ButtonGroup({ isCurrentlyCheckedOut, canEdit, isIssue, repositoryDefaultBranch, setEditMode }) {
+	const { refresh, copyPrLink } = useContext(PullRequestContext);
+
+	return (
+		<div className="button-group">
+			<CheckoutButtons {...{ isCurrentlyCheckedOut, isIssue, repositoryDefaultBranch }} />
```

Comment:
I'm a bit confused on how the `{...{a, b, c}}` part works. I'd have expected something like `{...[a, b, c]}` considering how the variables were already extracted in the function parameter.

Reply:
This syntax was already used in those components so I'll leave this for a follow up PR.